### PR TITLE
edit

### PR DESCRIPTION
**Pull request by Qlotto**

### CODE REVIEW

File: js/modal.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+$(document).ready(function(){
```
Comment (IlyaBass) [inline js/modal.js:1]: are you sure u need it?

File: article_page.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            </div>
+
+            <!-- <div class="blog__data">24.01.2023<a href="#" class="blog__edit">изменить</a></div> -->
+        </div>
+
+        <script src="./js/jquery-3.6.3.min.js"></script>
+        <script src="./js/modal.js"></script>
+        <script src="./js/fancybox.umd.js"></script>
+</body>
+</html>
```
Comment (IlyaBass) [inline article_page.html:44]: add empty line at the end of each file

File: article_page.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/article_page.css">
+    <link rel="stylesheet" href="./css/fancybox.css">
+    <title>article_page</title>
```
Comment (IlyaBass) [inline article_page.html:9]: name files like "article.html", not "article_page.html" etc.

Comment (IlyaBass) [reply article_page.html:9]: "_page" is redundant everywhere

File: article_page.html [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+</head>
+<body>
+    <div class="page-one">
+        <div class="nav">
```
Comment (IlyaBass) [inline article_page.html:13]: use nav tag

File: article_page.html [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+            <div class="nav__add"><a href="#">Главная</a></div>
+            <div class="nav__exit"><a href="#">Выход</a></div>
+        </div>
+
+        <div class="blog">
+            <div class="blog__name" data-fancybox data-src="#hidden-content">Название статьи<a href="#" class="blog__edit">изменить</a></div>
+            <div class="blog__description" data-fancybox data-src="#hidden-content">описание статьи описание статьи описание статьи<a href="#" class="blog__edit">изменить</a></div>
+            <div class="blog__categories" data-fancybox data-src="#hidden-content">Все<a href="#" class="blog__edit">изменить</a></div>
+
+            <div class="hidden-content" id="hidden-content">
```
Comment (IlyaBass) [inline article_page.html:23]: id must be unique and meaningful

File: article_page.html [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+                    <datalist id="list">
+                        <option value="Все">
+                        <option value="Мода">
+                        <option value="Не мода">
+                    </datalist>
+                    <input type="submit" value="Изменить">
+                </form>
+            </div>
+
+            <!-- <div class="blog__data">24.01.2023<a href="#" class="blog__edit">изменить</a></div> -->
```
Comment (IlyaBass) [inline article_page.html:37]: ?

File: index.html [hunk 7]
```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/style.css">
+    <title>login</title>
```
Comment (IlyaBass) [inline index.html:8]: why login if it's registration?

File: index.html [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+    <div class="page-one">
+        <div class="nav">
+            <a href="#">Регистрация</a>
+        </div>
+
+        <div class="login">
+            <form action="#" >
+                <input type="text" name="name" placeholder="Имя">
+                <input type="password" name="password" placeholder="Пароль">
+                <input type="submit" name="Login" value="Логин">
```
Comment (IlyaBass) [inline index.html:20]: why login?

File: main_page.html [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+        <div class="categories">
+            <div class="categories__all"><a href="#">Все</a></div>
+        </div>
+
+        <div class="blog">
+            <div class="blog__name"><a href="#">Название статьи</a></div>
+            <div class="blog__categories">Все</div>
+            <div class="blog__data">24.01.2023</div>
+            <div class="blog__description">описание статьи описание статьи описание статьи</div>
+        </div>
```
Comment (IlyaBass) [inline main_page.html:42]: how are you going to place plural items?

File: registration.html [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/registration.css">
+    <title>registration</title>
+</head>
+<body>
+    <div class="page-one">
+        <div class="nav">
+            <a href="#">Логин</a>
```
Comment (IlyaBass) [inline registration.html:13]: why login?

File: css/article_page.css [hunk 11]
```diff
@@ -0,0 +1,150 @@
+/* Указываем box sizing */
```
Comment (IlyaBass) [inline css/article_page.css:1]: you should place common lines of code to common css file
don't repeat yourself

File: index.html (hunk 7 above)
Comment (Qlotto) [reply index.html:8]: on this page people login

File: index.html (hunk 8 above)
Comment (Qlotto) [reply index.html:20]: on this page people login

File: registration.html (hunk 10 above)
Comment (Qlotto) [reply registration.html:13]: this is the login page

File: main_page.html (hunk 9 above)
Comment (Qlotto) [reply main_page.html:42]: do I need to make a list?

File: css/article_page.css [hunk 12]
```diff
@@ -0,0 +1,149 @@
...
+*,
+*::before,
```
Comment (IlyaBass) [inline css/article_page.css:3]: same, delete _page for each css file

Comment (IlyaBass) [inline css/article_page.css:3]: you should place common lines of code to common css file
don't repeat yourself
you didn't resolve this problem

File: css/reset.css [hunk 13]
```diff
@@ -0,0 +1,81 @@
...
+
+/* Удаляем все анимации и переходы для людей, которые предпочитай их не использовать */
+@media (prefers-reduced-motion: reduce) {
+  * {
+    animation-duration: 0.01ms !important;
+    animation-iteration-count: 1 !important;
+    transition-duration: 0.01ms !important;
+    scroll-behavior: auto !important;
+  }
+}
```
Comment (IlyaBass) [inline css/reset.css:81]: add empty line!!!!!

File: css/style.css [hunk 14]
```diff
@@ -0,0 +1,18 @@
...
+    margin-top: 40px;
+    margin-left: 50px;
+}
+
+.nav a{
+    color: white;
+    letter-spacing: 0.5px;
+    text-decoration: none;
+}
+
```
Comment (IlyaBass) [inline css/style.css:11]: why it's style.css if it supposed to be login.css

File: main.html [hunk 15]
```diff
@@ -0,0 +1,78 @@
...
+            </div>
+            <div class="nav__exit"><a href="#">Выход</a></div>
+        </nav>
+
+        <div class="categories">
+            <div class="categories__all"><a href="#">Все</a></div>
+        </div>
+
+    <div class="blog-system">
+        <div class="blog">
```
Comment (IlyaBass) [inline main.html:39]: do normal naming.. why each blog item is blog??

File: main.html [hunk 16]
```diff
@@ -0,0 +1,78 @@
...
+
+        <div class="blog">
+            <div class="blog__name"><a href="#">Название статьи</a></div>
+            <div class="blog__categories">Все</div>
+            <div class="blog__data">24.01.2023</div>
+            <div class="blog__description">описание статьи описание статьи описание статьи</div>
+        </div>
+    </div>
+        <script src="./js/jquery-3.6.3.min.js"></script>
+        <script src="./js/fancybox.umd.js"></script>
```
Comment (IlyaBass) [inline main.html:75]: why scripts are in div?

File: css/article.css [hunk 17]
```diff
@@ -0,0 +1,65 @@
...
+
+a:hover{
+  color: yellow;
+}
+/* ssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss */
+
+.hidden-content{
+  display: none;
+}
+
```
Comment (IlyaBass) [inline css/article.css:57]: add this to common css file.
don't u see that it repeats in some files??

File: main.html [hunk 18]
```diff
@@ -0,0 +1,79 @@
...
+            <div class="nav__exit"><a href="#">Выход</a></div>
+        </nav>
+
+        <div class="categories">
+            <div class="categories__all"><a href="#">Все</a></div>
+        </div>
+
+    <div class="blog">
+        <div class="blog__item">
+            <div class="blog__item__name"><a href="#">Название статьи</a></div>
```
Comment (IlyaBass) [inline main.html:40]: read about BEM mothodology and fix naming

File: css/article.css (hunk 17 above)
Comment (IlyaBass) [reply css/article.css:57]: don't repeat yourself!!!!

**Pull request merged**